R/RSF/build_tarballs.jl

```diff
@@ -0,0 +1,1888 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "RSF"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/ahay/src.git", "3a10302c63c69317df3f6f741347e7f77a2f3d58")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+
+# install_license
+install_license src/COPYING.txt
+
+# Build
+cd src/
+
+./configure --prefix=${prefix} CC=$(which gcc) CXX=$(which g++) CAIROPATH=${includedir}/cairo FFMPEGPATH=${includedir}/libavcodec/ AR=$(which ar) CPPPATH=${includedir} LIBPATH=${libdir} BLAS=openblas
```

Comment:
Why `CC=$(which gcc)`, `CXX=$(which g++)` and `AR=$(which ar)` (the former two are also wrong)?  The corresponding environment variable are already set to the right values.

Reply:
The configure is using `scons` which doesn't find any of the compiles from the env vars unless specified here so all three `gcc/g++/ar` are needed or it fails.